packages/commonwealth/server/passport/magic.ts

```diff
@@ -90,6 +90,14 @@ async function createMagicAddressInstances(
               oauth_provider: oauthProvider,
             },
           },
+          {
```

Comment:
@dillchen should we emit both CommunityJoined and SSOLinked from this magic action or just SSOLinked? I'm keeping both to follow our current understanding of new address record => joining a community